### DISCUSSION TITLE
9.2.4 Release

### DISCUSSION THREAD

**Discussion by Donnnno** [Arcticons-Team/Arcticons#2103] [Announcements]
🎉 243 new and updated icons!
✨ Updates to make sure many apps work again.
🚀 Moto Launcher support.
🌐 The beginning of [a new wiki](https://github.com/Arcticons-Team/Arcticons/wiki/) page for all your Arcticons questions!
🔥 9240 icons in total!

-----------------

## Complete overview
* Add Update Dashboard by @Kaiserdragon2 in https://github.com/Arcticons-Team/Arcticons/pull/2047
* fix Copy paste error by @Kaiserdragon2 in https://github.com/Arcticons-Team/Arcticons/pull/2048
* fix link by @Kaiserdragon2 in https://github.com/Arcticons-Team/Arcticons/pull/2049
* more pngs by @Kaiserdragon2 in https://github.com/Arcticons-Team/Arcticons/pull/2061
* more png by @Kaiserdragon2 in https://github.com/Arcticons-Team/Arcticons/pull/2062
* Pngs by @Kaiserdragon2 in https://github.com/Arcticons-Team/Arcticons/pull/2063
* more png by @Kaiserdragon2 in https://github.com/Arcticons-Team/Arcticons/pull/2066
* try solving that files with _ at beginning aren't included in pages by @Kaiserdragon2 in https://github.com/Arcticons-Team/Arcticons/pull/2067
* fix filname by @Kaiserdragon2 in https://github.com/Arcticons-Team/Arcticons/pull/2068
* 110+ Icons by @Kaiserdragon2 in https://github.com/Arcticons-Team/Arcticons/pull/2051
* Readme & Appfilter fixes by @TotallyAvailable in https://github.com/Arcticons-Team/Arcticons/pull/2046
* Fix appfilter.xml by @willycast in https://github.com/Arcticons-Team/Arcticons/pull/2050
* Add 25 new icons by @Kartanovicius in https://github.com/Arcticons-Team/Arcticons/pull/2064
* Update updatable.js by @Kaiserdragon2 in https://github.com/Arcticons-Team/Arcticons/pull/2078
* copy only required appfilter by @Kaiserdragon2 in https://github.com/Arcticons-Team/Arcticons/pull/2079
* 5 more Icons by @Kaiserdragon2 in https://github.com/Arcticons-Team/Arcticons/pull/2073
* Wiki workflow by @Kaiserdragon2 in https://github.com/Arcticons-Team/Arcticons/pull/2077
* Added 5 Icons by @TheAngryTurkey in https://github.com/Arcticons-Team/Arcticons/pull/2071
* Added contribution by @TheAngryTurkey in https://github.com/Arcticons-Team/Arcticons/pull/2070
* Mapping InstaFlow (yet another IG client) to Instagram by @dazewell in https://github.com/Arcticons-Team/Arcticons/pull/2065
* Revert "Added 5 Icons" by @Donnnno in https://github.com/Arcticons-Team/Arcticons/pull/2083
* Add 10 icons by @Npepperlinux in https://github.com/Arcticons-Team/Arcticons/pull/2056
* add counter to updatable site by @Kaiserdragon2 in https://github.com/Arcticons-Team/Arcticons/pull/2086
* add packagename display and copy function for packagename by @Kaiserdragon2 in https://github.com/Arcticons-Team/Arcticons/pull/2087
* add sorting remove unused function by @Kaiserdragon2 in https://github.com/Arcticons-Team/Arcticons/pull/2088
* try having combined appfilter in main by @Kaiserdragon2 in https://github.com/Arcticons-Team/Arcticons/pull/2090
* Fix Sorting arrow move request.txt by @Kaiserdragon2 in https://github.com/Arcticons-Team/Arcticons/pull/2091
* Update appfilter.xml by @Kaiserdragon2 in https://github.com/Arcticons-Team/Arcticons/pull/2085
* Added 8 icons, and appfilter entries by @joelchrono12 in https://github.com/Arcticons-Team/Arcticons/pull/2093
* 8 new icons by @albertjohnsonpc in https://github.com/Arcticons-Team/Arcticons/pull/2092
* Adds 5 new icons by @trautb in https://github.com/Arcticons-Team/Arcticons/pull/2058
* Update appfilter.xml for 2 apps by @Npepperlinux in https://github.com/Arcticons-Team/Arcticons/pull/2089
* audi_bkk_service app by @ayecptn in https://github.com/Arcticons-Team/Arcticons/pull/2054
* Ma French Bank by @GuitareCouture in https://github.com/Arcticons-Team/Arcticons/pull/2057
* Add icon: wasteof for android by @Cequallium in https://github.com/Arcticons-Team/Arcticons/pull/2094
* Add Chrono icon by @TheLastZombie in https://github.com/Arcticons-Team/Arcticons/pull/2084
* 2 New Icon by @onuronsekiz in https://github.com/Arcticons-Team/Arcticons/pull/2060
* Update Home.md by @Donnnno in https://github.com/Arcticons-Team/Arcticons/pull/2096
* Change String for launcher support remove unused icons by @Kaiserdragon2 in https://github.com/Arcticons-Team/Arcticons/pull/2076
* appfilter updates by @Donnnno in https://github.com/Arcticons-Team/Arcticons/pull/2100
* some lazy icons by @Donnnno in https://github.com/Arcticons-Team/Arcticons/pull/2101
* Icons from request by @Kaiserdragon2 in https://github.com/Arcticons-Team/Arcticons/pull/2098
* have the update site show the icon by @Kaiserdragon2 in https://github.com/Arcticons-Team/Arcticons/pull/2102

## New Contributors
* @TheAngryTurkey made their first contribution in https://github.com/Arcticons-Team/Arcticons/pull/2071
* @Npepperlinux made their first contribution in https://github.com/Arcticons-Team/Arcticons/pull/2056
* @albertjohnsonpc made their first contribution in https://github.com/Arcticons-Team/Arcticons/pull/2092
* @GuitareCouture made their first contribution in https://github.com/Arcticons-Team/Arcticons/pull/2057
* @Cequallium made their first contribution in https://github.com/Arcticons-Team/Arcticons/pull/2094

**Full Changelog**: https://github.com/Arcticons-Team/Arcticons/compare/9.0.0...9.2.4

<hr /><em>This discussion was created from the release <a href='https://github.com/Arcticons-Team/Arcticons/releases/tag/9.2.4'>92.4 Release</a>.</em>

**Comment by Suburbanno** [comment]
92.4 😶‍🌫️

**Reply by Donnnno** [reply]
uhhhhh

**Reply by Donnnno** [reply]
fixed it!
